### ISSUE TITLE
Can't install master version

### ISSUE THREAD

**Issue by jclsn** [maxbec/hassio-diyHue#47]
The installation fails with 

```
The command '/bin/bash -o pipefail -c mv /diyhue/BridgeEmulator/web-ui/ ./web-ui/' returned a non-zero code: 1
```

You can install the beta, but I couldn't get it to connect to the Hue app either. Would be great if you could update the addons and the documentation.

**Comment by TheMeek** [comment]
same with me its bveen quite some time sinche i have been able to

**Comment by Sergiodiaz53** [comment]
Same here!

**Comment by jclsn** [comment]
Yeah someone needs to fork this repo and update the Dockerfile to look like the one from the beta mostly!

Gruß,
Jan Claußen

**Comment by Sergiodiaz53** [comment]
Well I think there are more problems than that. 
I have installed the beta and Hue Sync is not working:

`2022-04-30 12:23:03,570 - flaskUI.v2restapi - INFO - start hue entertainment
10.0.0.27 - - [30/Apr/2022 12:23:04] "PUT /clip/v2/resource/entertainment_configuration/7bbfea4b-7d25-5874-bf79-7cf88e871931 HTTP/1.1" 200 -
2022-04-30 12:23:04,648 - services.stateFetch - INFO - start lights sync
10.0.0.27 - - [30/Apr/2022 12:23:04] "GET /clip/v2/resource/entertainment_configuration HTTP/1.1" 200 -
10.0.0.15 - - [30/Apr/2022 12:23:08] "GET /api/032d6f7cc86e11eca902e4b97a4e0ceb/groups/0 HTTP/1.1" 200 -
10.0.0.27 - - [30/Apr/2022 12:23:08] "{"action":"start"}GET /auth/v1 HTTP/1.1" 405 -
10.0.0.15 - - [30/Apr/2022 12:23:13] "GET /api/032d6f7cc86e11eca902e4b97a4e0ceb/groups/0 HTTP/1.1" 200 -
10.0.0.27 - - [30/Apr/2022 12:23:14] "GET / HTTP/1.1" 302 -
10.0.0.27 - - [30/Apr/2022 12:23:14] code 400, message Bad request syntax ('{"action":"start"}')
10.0.0.27 - - [30/Apr/2022 12:23:14] "None /auth/v1 HTTP/0.9" HTTPStatus.BAD_REQUEST -`

**Comment by jclsn** [comment]
Might be fixed in the latest release

**Comment by jclsn** [comment]
I could fix the installation

https://github.com/maxbec/hassio-diyHue/pull/49

but no idea how the app works. I can't connect when pressing the link button.,

I am also discussing it here https://github.com/diyhue/diyHue/issues/809

**Comment by Mevel** [comment]
I would like to invite you to the official diyhue-hassio addon repo found here

https://github.com/diyhue/hassio-addon

it is based upon this repo and will be maintained officially.

**Comment by jclsn** [comment]
@Mevel Thanks, I will try it at once

**Comment by jclsn** [comment]
@Mevel: Can you please also update the documentation? I think it would be smart to add detailed configuration steps.

**Comment by jclsn** [comment]
It is not installing for me. It's been over 10 minutes

**Comment by chillkiller** [comment]
@jclsn its takes a lot of time to install but I got it work

otherwise perhaps there is more issues on your system that is blocking your diyhue install

**Comment by jclsn** [comment]
@chillkiller It took ages, but it finished now. I still can't connect to the Hue app though. It says "Check your internet connection". I have taken the MAC address of the Pi's ethernet adapter. Is that wrong?

**Comment by chillkiller** [comment]
yeah,its the ip of your pi adapter, but every time you install it you need to delete the old cert.perm

**Comment by jclsn** [comment]
Yeah, I had and old configuration under ``config/diyHue``. I deleted it now and I could connect

**Comment by jclsn** [comment]
Now I have to integrate HA back into diyHue I guess? xD

**Comment by Mevel** [comment]
@chillkiller thank you for the support! Glad you got your instance running.

@jclsn if you need further assistance join the diyhue slack on

Joinslack.diyhue.org

Feel free to contact me

**Comment by jclsn** [comment]
@chillkiller What haha. I was trying this nasty guide https://developers.home-assistant.io/docs/auth_api/#long-lived-access-token

**Comment by jclsn** [comment]
@Mevel Noo, yet another messenger. I have Telegram, Signal, Matrix, Discord, IRC, Teams and now Slack..

**Comment by Mevel** [comment]
For now slack is the most active but we are starting discord too.
